build/media/libraries/redcore/media/redcore/css/component.bs3.css

```diff
@@ -14,9 +14,9 @@
   font-size: 14px;
   line-height: 1.42857143;
   color: #333333;
-  background-color: inherit;
-  /*
- * Bootstrap v3.3.5 (http://getbootstrap.com)
+  background-color: #fff;
```

Comment:
I am worrying about all these changes that .less file is making on the .css files without you changing it personally. Can I ask you to go through these changes and put them into .less file as well. What may happen is that developer updated .css file without changing .less file. So we dont end up with overwriting our own fixes.

Reply:
Sure. I can do all the changes that we need, but I swear that I'm only editing less file.
I only added this at the end of file // Custom permissions div
#permissions .container-fluid > div { 
  width: 100%; 
} 
Gulp made the rest.

Reply:
so, can you add the changes from css file to the less file, since it seems that we have some of the changes there that were not put into less file, and I dont want us to lose them.